using-mobile-extensions/adobe-target/README.md

```diff
@@ -183,64 +182,223 @@ prefetchMboxesList.add(prefetchRequest2);
+                                                    profileParameters:nil
+                                                              product:product2
+                                                                order:order2];
 
 // Creating Prefetch Objects
-ACPTargetPrefetchObject *prefetch1 = [ACPTargetPrefetchObject prefetchObjectWithName:@"logo" mboxParameters:mboxParameters1];
-prefetch1.productParameters = productParameters1;
-prefetch1.orderParameters = orderParameters1;
+ACPTargetPrefetchObject *prefetch1 = [ACPTargetPrefetchObject targetPrefetchObjectWithName:@"logo"
+                                                                         targetParameters:targetParameters1];
 
-ACPTargetPrefetchObject *prefetch2 = [ACPTargetPrefetchObject prefetchObjectWithName:@"buttonColor" mboxParameters:mboxParameters2];
-prefetch2.productParameters = productParameters2;
-prefetch2.orderParameters = orderParameters2;
+ACPTargetPrefetchObject *prefetch2 = [ACPTargetPrefetchObject targetPrefetchObjectWithName:@"buttonColor"
+                                                                         targetParameters:targetParameters2];
 
 // Creating prefetch Array
 NSArray *prefetchArray = @[prefetch1,prefetch2];
 
-// Creating Profile parameters
+// Creating Target parameters
+NSDictionary *mboxParameters = @{@"status":@"progressive"};
 NSDictionary *profileParameters = @{@"age":@"20-32"};
+ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
+ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
+ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:mboxParameters
+                                                    profileParameters:profileParameters
+                                                              product:product
+                                                                order:order];
+
 
 // Target API Call
-[ACPTarget prefetchContent:prefetchArray withProfileParameters:profileParameters callback:^(BOOL isSuccess){
-       // do something with the Boolean result
+[ACPTarget prefetchContent:prefetchArray withParameters:targetParameters callback:^(NSError * _Nullable error){
+       // do something with the callback response
 }];
 ```
 {% endtab %}
 {% endtabs %}
 
+### Using TargetParameters, TargetOrder and TargetProduct object
+
+Using `TargetParameters`, you can encapsulate various parameters (namely mboxParameters, profileParameters, orderParameters and productParameters) for easy use
+With `TargetOrder`, you can encapsulate the order parameters and use it in TargetParameters.
+With `TargetProduct`, you can encapsulate the product parameters and use it in TargetParameters.
+
+
+
+**Merging behavior of parameters passed in API's  with parameters passed in TargetPrefetch/TargetRequest Object**
```

Comment:
Please reword the title so that it reads as follows:

"**Merging the behavior of parameters that are passed in the APIs with parameters that are passed in the TargetPrefetch/TargetRequest Object**

Reply:
Changed to "Merge behavior of parameters that are passed in the APIs with parameters that are passed in the TargetPrefetch/TargetRequest Object"